modules/kafka/kafka.go

```diff
@@ -125,12 +152,78 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 	return &KafkaContainer{Container: container, ClusterID: clusterID}, nil
 }
 
-func WithClusterID(clusterID string) testcontainers.CustomizeRequestOption {
-	return func(req *testcontainers.GenericContainerRequest) error {
-		req.Env["CLUSTER_ID"] = clusterID
+func trimListeners(listeners []KafkaListener) {
+	for i := 0; i < len(listeners); i++ {
+		listeners[i].Name = strings.ToUpper(strings.Trim(listeners[i].Name, " "))
+		listeners[i].Ip = strings.Trim(listeners[i].Ip, " ")
+		listeners[i].Port = strings.Trim(listeners[i].Port, " ")
+	}
+}
 
-		return nil
+func validateListeners(listeners []KafkaListener) error {
```

Comment:
Let's add unit tests for this function 🙏

Reply:
Will do